### DISCUSSION TITLE
Deleted Email

### DISCUSSION THREAD

**Discussion by yoshiyoyoi** [community/community#73083] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I deleted the gmail account that I used for my github account.
I need to get back into my account, but I can't get back in now cause the old gmail account does not exist anymore.
I forgot the email was used on the account.

**Comment by TankEngine-ish** [comment, marked as answer]
I think you should contact github support directly - [here](https://support.github.com/contact/account)

**Comment by gitstua** [comment]
Hi there! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Comment by IliusSagar** [comment]
Contact Github Support Team Request.

**Comment by rb2938112-sys** [comment]
My Gmail make koi sa bhi code nahin a raha hai try again cover substan de raha hai
"Personal info edited by staff"
